app/2.2.x/kong-user.md

```diff
@@ -34,7 +34,7 @@ configuration property.
 
 ## Run Kong Gateway as a non-root user
 
-1. Switch to the `kong` user and group:
+1. Switch to the `kong` user:
     ```sh
     $ su kong
     ```
```

Comment:
I think there needs to be a line here to introduce the possibility of using a custom user, for example:

You can also switch to a custom non-root user. This will require an extra setup step.